specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.2/securitydomain.json

```diff
@@ -0,0 +1,244 @@
+                    }
+                }
+            },
+            "post": {
+                "tags": [
+                    "HSMSecurityDomain"
+                ],
+                "operationId": "HSMSecurityDomainUpload",
+                "description": "Request Security domain restore operation",
+                "parameters": [
+                    {
+                        "in": "body",
+                        "name": "security_domain",
+                        "required": true,
+                        "schema": {
+                            "type": "object",
+                            "properties": {
+                                "value": {
+                                    "type": "string"
+                                }
+                            }
+                        }
+                    }
+                ],
+                "responses": {
+                    "202": {
+                        "description": "Security domain restore operation started",
+                        "headers": {
+                            "Retry-After": {
+                                "description": "The recommended number of seconds to wait before calling the URI specified in Azure-AsyncOperation.",
+                                "type": "integer"
+                            },
+                            "Azure-AsyncOperation": {
+                                "description": "The URI to poll for completion status.",
+                                "type": "string"
+                            }
+                        },
+                        "schema": {
+                            "type": "object",
+                            "properties": {
+                                "status": {
+                                    "type": "string"
+                                },
+                                "status_details": {
+                                    "type": "string"
+                                }
+                            }
+                        }
+                    },
+                    "default": {
+                        "description": "Key Vault error response describing why the operation failed.",
+                        "schema": {
+                            "$ref": "common.json#/definitions/KeyVaultError"
+                        }
+                    }
+                },
+                "x-ms-examples": {
+                    "Security domain restore operation": {
+                        "$ref": "./examples/securitydomainrestoreoperation-example.json"
+                    }
+                }
+            }
+        },
+        "/securitydomain/upload/pending": {
+            "get": {
+                "tags": [
+                    "HSMSecurityDomain"
+                ],
+                "operationId": "HSMSecurityDomainUploadPending",
+                "description": "Get Security domain restore operation status",
```

Comment:
Consistency. If we keep calling this operation `upload`, use Get Security Domain upload operation status.